src/libxtp/dftengine/dftengine.cc

```diff
@@ -380,6 +383,31 @@ namespace votca {
                 // DFT AOOverlap matrix
                 _dftAOoverlap.Fill(_dftbasis);
                 CTP_LOG(ctp::logDEBUG, *_pLog) << ctp::TimeStamp() << " Filled DFT Overlap matrix of dimension: " << _dftAOoverlap.Dimension() << flush;
+
+                if (_with_periodic_embedding) {
```

Comment:
This needs to be refactored into a proper function

Reply:
Agreed, but it depends on the yet to be written embedding object and the interfaces to that. Should be done in a separate feature branch.

Reply:
I would not like this to be in the dev branch, before it is complete